module1/homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,25 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int fCurrent = 1, f1 = 1, f2 = 1;
+    
+    if (sequence == 0)
+      return 0;
+    
+    for (auto iter = 3 ; iter <= sequence ; iter++) {
+      fCurrent = f1 + f2;
+      f1       = f2;
+      f2       = fCurrent;
+    }
+    
+    return fCurrent;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence<0)
```

Comment:
```suggestion
    if(sequence < 0)
```
Remember of coding style. You used spaces before, they are missing here.